ehitchhiking-ui/src/app/pages/registration/registration.component.sass

```diff
@@ -70,7 +70,7 @@ div .exadel-login
   display: inline-block
   margin: 0 5% 0 0
   text-align: center
-  width: 1.2rem
+  width: $default-offset * 1.2
```

Comment:
you should not compute width using $default-offset. $default-offset should be used for paddings, margings

Reply:
done